src/clients/docs_generate.zig

```diff
@@ -473,11 +473,11 @@ const Generator = struct {
             \\ID and replica addresses are both chosen by the system that
             \\starts the TigerBeetle cluster.
             \\
-            \\Clients are thread-safe. But for better
-            \\performance, a single instance should be shared between
-            \\multiple concurrent tasks.
+            \\Clients are thread-safe but serialize requests internally.
```

Comment:
Any better word that "serialize" here? I want to convey that each client is thread-safe, but sequential, serialize has that sense, but also half of the people reading that would thing about JSON serialization or something...

Reply:
"but order requests serially internally"?